### PR TITLE
sc761, remove starkware derived key

### PR DESCRIPTION
**Pull request by jar-o**
https://app.shortcut.com/chronicle-labs/story/761/remove-starkware-signing-from-omnia

- [x] removes starkware logic 
- [x] removes starkware deps
- [x] adds Makefile + updates README

### CODE REVIEW

File: lib/config.sh [hunk 1]
```diff
@@ -132,25 +131,6 @@ importEthereumEnv () {
...
-importStarkwareEnv() {
-	verbose "Importing Starkware signature keys"
-	local _starkdata
-	local _enthropy
-	local _seed
-
-	STARK_CLI="${STARK_CLI:-stark_cli.py}"
-	export STARK_CLI
-
-	_starkdata="537461726b4b657944657269766174696f6e" #StarkKeyDerivation
```
Comment (jar-o) [inline lib/config.sh:144]: buh bye

File: test/units/feed.sh [hunk 2]
```diff
@@ -89,4 +87,4 @@ transportPublish() {
 }
 export -f transportPublish
 assert "readSourcesAndBroadcastAllPriceMessages should run without calling for transportPublish on empty JSON object source msg" \
-	run readSourcesAndBroadcastAllPriceMessages
```
Comment (teghnet) [inline test/units/feed.sh:92]: :smiley:

**Comment by teghnet** [comment]
I think we need to remove stark stuff drom tests: `test/e2e/message.go` etc.

File: test/units/feed.sh (hunk 2 above)
Comment (jar-o) [reply test/units/feed.sh:92]: heh, tabs yo

**Review by madumas — APPROVED** [review]

**Pull request merged**